lib/devise/models/confirmable.rb

```diff
@@ -169,7 +175,6 @@ def skip_reconfirmation!
         # in models to map to a nice sign up e-mail.
         def send_on_create_confirmation_instructions
           send_confirmation_instructions
-          skip_reconfirmation!
```

Comment:
Thanks for the reference ! I didn't remove that line because one spec will fail. Instead I moved it above.